Widgets/widget.css

```diff
@@ -2,11 +2,12 @@ body {
   font-family: 'Cantarell';
   color:white;
   background-color: #194d6c;
+  font-weight:700;
```

Comment:
As long as you don't include the 400 size font, I don't think anything *can't* be bold. But yeah, worth testing